traffic_ops/app/db/migrations/2022032914235900_add_server_profile_table.up.sql

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
`2022040623082100_server_config_update.down.sql` and `2022040623082100_server_config_update.up.sql` were added in #6730. so now the timestamp in the filename of the layered profile up/down migrations should be increased from `2022032914235900` to some value greater than `2022040623082100` so it is the last migration in the list when #6544 gets merged.